caveclient/tools/table_manager.py

```diff
@@ -374,10 +378,10 @@ def __attrs_post_init__(self):
                 tn: filter_empty(
                     attrs.asdict(
                         self,
-                        filter=lambda a, v: is_list_like(v) == False
+                        filter=lambda a, v: not is_list_like(v)
                         and v is not None
-                        and a.metadata.get("is_bbox", False) == False
-                        and a.metadata.get("is_meta", False) == False
+                        and a.metadata.get("is_bbox", False) == False  # noqa E712
```

Comment:
could just fix these... but unsure whether this get could return a string, and then things get weird... so i opted to just tell it to shut up for these, and hopefully in the future we can just follow the linting